bin/i18n/utils/sync_out_base.rb

```diff
@@ -19,7 +19,9 @@ def process(_language)
       end
 
       def languages
-        @languages ||= PegasusLanguages.all
+        @languages ||= I18nScriptUtils.cdo_languages.select do |cdo_language|
+          cdo_language[:locale_s] != I18nScriptUtils::SOURCE_LOCALE
+        end
```

Comment:
Could you add a comment in the code explaining what is the purpose of the `languages` method? and explaining what it returns. I'm concerned other developers may assume it returns all languages.